File: SmartVitals_Server/SmartVitals_Server/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel.DataAnnotations;
using SmartVitals.Services.Models;

namespace SmartVitals_Server.Models
{
    public class Comment
    {
        [Key]
        public int CommentId { get; set; }
        public int TestId { get; set; }
        //public Test Test { get; set; }
        public String Message {get;set;}
        public String UserName { get; set; }
        public DateTime Date { get; set; }
    }
}

File: SmartVitals_App/SmartVitals_App/src/usf/smartvitals/json/Models/Comment.java
package usf.smartvitals.json.Models;

import com.google.gson.annotations.SerializedName;

public class Comment
{
	@SerializedName("CommentId")
    public int CommentId;
    
	@SerializedName("TestId")
	public int TestId;
	
	@SerializedName("Test")
    public Test Test;
	
	@SerializedName("Comment")
    public String Message;
	
	@SerializedName("UserName")
    public String UserName;
	
	@SerializedName("Date")
    public String Date;
}
